### DISCUSSION TITLE
YAML from another repo

### DISCUSSION THREAD

**Discussion by botkero** [fralau/mkdocs-macros-plugin#145] [Q&A]
hello,
I would like to know if it is somehow possible to specify a repo link as source for the yaml file.

```
  - macros:
      include_yaml:
        - repo_url
 ```

**Comment by redtide** [comment]
I'm also interested on this, but probably it could add some level of undesired complexity (online connection handling). Maybe a python script/hook that downloads/updates the file before the process?
